gnovm/tests/package_test.go

```diff
@@ -52,6 +51,102 @@ func TestPackages(t *testing.T) {
 	for _, pkgPath := range pkgPaths {
 		testDir := testDirs[pkgPath]
 		t.Run(pkgPath, func(t *testing.T) {
+			if pkgPath == "gno.land/p/demo/avl" {
```

Comment:
These ones seem to be failing because they're attempting to run filetests, but this test function doesn't support filetests.

And I don't think it should. We should test the `examples` directory using `go run github.com/gnolang/gno/gnovm/cmd/gno test ./examples/...`, as the current examples.yml does.

Reply:
I tend to agree with @thehowl on this 👍

Reply:
shall then remove this test and create a new one mimicking what `examples.yml` is doing but using `go test`?

Reply:
Let's also make sure `TestPackages` only tests the standard libraries. We can also rename it to `TestStdlibs`

Reply:
leaving this change to following iterations.